First solution

Reyti opened the pull request:
- [DEMO LINK](https://reyti.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://reyti.github.io/layout_moyo-header/report/html_report/)

etojeDenys commented on src/index.html:
```diff
@@ -7,8 +7,51 @@
+          <li class="nav__item">
+            <a class="link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="link"
+            href="#laptops"
+            data-qa="hover"
+            >Laptops & Computers</a>
+          </li>
```
```suggestion
          <li class="nav__item">
            <a
              class="link"
              href="#laptops"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
```

Reyti replied:
**Added links to the PR desc**

- [DEMO LINK](https://reyti.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://reyti.github.io/layout_moyo-header/report/html_report/)

etojeDenys reviewed (changes requested):
provide the links in PR description

loralevitska reviewed (changes requested):
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

Reyti commented on src/index.html:
```diff
@@ -7,8 +7,51 @@
+            href="#laptops"
+            data-qa="hover"
+            >Laptops & Computers</a>
+          </li>
```
Fixed now.

Reyti commented:
Fixed positioning of a link

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -7,8 +7,53 @@
     <link rel="stylesheet" href="./style.css">
+    <!-- fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+      >
```
```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com">
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <!-- fonts -->
```
```suggestion
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -7,8 +7,53 @@
+            <a class="link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="link"
+            href="#laptops"
+            data-qa="hover"
+            >
+            Laptops & Computers
```
```suggestion
            <a
              class="link"
              href="#laptops"
              data-qa="hover"
            >
              Laptops & Computers
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -7,8 +7,53 @@
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
```
Add a blank line between the links for better readability
```suggestion
          </li>
          
          <li class="nav__item">
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -7,8 +7,53 @@
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="link is-active" href="#apple">Apple</a>
```
Need to fix naming classes
```suggestion
            <a class="nav__link is-active" href="#apple">Apple</a>
```

volodymyr-soltys97 reviewed (changes requested):
Good job 🔥 
Let's improve your code

Reyti commented:
1. Added a blank line between the links for better readability
2. Deleted unnecessary comments.
3. Added a space to three elements inside the link.
4. Changed class name of link inside a nav element to nav__link for better usability.
5. Minor fixes over the font links.

TarasHoliuk reviewed (approved):
Well done :+1:
